src/utils/isCustomIdentPropertyAnimationName.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Check value is a custom ident
+ *
+ * @param {string} value
+ * @return {boolean} If `true`, value is a custom ident
+ */
+
+// https://developer.mozilla.org/ru/docs/Web/CSS/animation-name
+const knownValues = new Set([
+  "none", "inherit", "initial", "unset",
+])
+
+export default function (value) {
+  if (knownValues.has(value.toLowerCase())) { return false }
```

Comment:
You can shorten this by just doing `return knownValues.has(value.toLowerCase())`, right?